src/System.Linq.Expressions/src/System/Linq/Expressions/DynamicExpression.cs

```diff
@@ -1172,29 +1076,29 @@ public static DynamicExpression Dynamic(CallSiteBinder binder, Type returnType,
             return MakeDynamic(binder, returnType, args);
         }
 
-        private static DynamicExpression MakeDynamic(CallSiteBinder binder, Type returnType, ReadOnlyCollection<Expression> args)
+        private static DynamicExpression MakeDynamic(CallSiteBinder binder, Type returnType, ReadOnlyCollection<Expression> arguments)
```

Comment:
Why this rename? I don't see any value of args->arguments. 'args' is popular and everyone understands it.